fn_ansible/fn_ansible/util/config.py

```diff
@@ -0,0 +1,25 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate a default configuration-file section for fn_ansible"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+    config_data = u"""[fn_ansible]
+#playbook_dir=</full/path/to/your/playbook/directory>
```

Comment:
* No need to have all these values 'commented out' by default
* We can remove the `#`s